### PR TITLE
Shedule task

### PR DESCRIPTION
**Pull request by Andrii1000**

### CODE REVIEW

File: library_service/celery.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+app = Celery("library_service")
+
+app.config_from_object("django.conf:settings", namespace="CELERY")
+
+app.autodiscover_tasks()
+
+
+@app.task(bind=True, ignore_result=True)
+def debug_task(self):
+    print(f'Request: {self.request!r}')
```
Comment (AntonKorinchuk) [inline library_service/celery.py:16]: Add empty line

File: payment/tasks.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from celery import shared_task
+
+
+@shared_task
+def checking_stripe_session():
```
Comment (AntonKorinchuk) [inline payment/tasks.py:5]: Add empty line

**Review by AntonKorinchuk — APPROVED** [review]

File: borrowing/tasks.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                       f"Status: {status}")
+            try:
+                send_message(message)
+            except Exception as e:
+                pass
+    else:
+        try:
+            send_message("No active borrowings today!")
+        except Exception as e:
+            pass
```
Comment (SuskyiVolodymyr) [inline borrowing/tasks.py:29]: no empty line

File: library_service/__init__.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from .celery import app as celery_app
+
+__all__ = ("celery_app",)
```
Comment (SuskyiVolodymyr) [inline library_service/__init__.py:3]: no empty line

File: library_service/celery.py (hunk 1 above)
Comment (SuskyiVolodymyr) [inline library_service/celery.py:16]: no empty line

File: library_service/settings.py [hunk 5]
```diff
@@ -160,3 +161,17 @@
...
+CELERY_TASK_SERIALIZER = "json"
+CELERY_RESULT_SERIALIZER = "json"
+CELERY_TIMEZONE = "UTC"
+
+CELERY_BEAT_SCHEDULE = {
+    "check_overdue_borrowings": {
+        "task": "borrowing.tasks.check_overdue_borrowings",
+        "schedule": timedelta(days=1),
+    },
+}
```
Comment (SuskyiVolodymyr) [inline library_service/settings.py:177]: no empty line

**Review by SuskyiVolodymyr — CHANGES_REQUESTED** [review]

File: library_service/settings.py (hunk 5 above)
Comment (Andrii1000) [reply library_service/settings.py:177]: same have

File: library_service/celery.py (hunk 1 above)
Comment (Andrii1000) [reply library_service/celery.py:16]: change in last commit

File: library_service/__init__.py (hunk 4 above)
Comment (Andrii1000) [reply library_service/__init__.py:3]: same are

File: borrowing/tasks.py (hunk 3 above)
Comment (Andrii1000) [reply borrowing/tasks.py:29]: here is

**Pull request closed without merging**